delta/serving/base_frozen_model.py

```diff
@@ -99,6 +99,24 @@ def init_session(self, model, gpu_str):
       self._graph = utils.load_frozen_graph(frozen_graph)
       self._sess = tf.Session(graph=self._graph, config=config)
 
+  def inspect_ops(self):
+    for op in self._graph.get_operations():
+      logging.info(op.name)
+
+  def debug(self):
+    feed_dict = self.get_test_feed_dict()
```

Comment:
self.get_test_feed_dict() this not common for all fronzen_model class.
should be self.feed_dict().